osu.Server.Queues.ScoreStatisticsProcessor/Commands/Performance/Scores/UpdateAllScoresByUserCommand.cs

```diff
@@ -78,14 +77,10 @@ protected override async Task<int> ExecuteAsync(CancellationToken cancellationTo
                 if (userIds.Length == 0)
                     break;
 
-                await ProcessPartitioned(userIds, async userId =>
+                await ProcessPartitioned(userIds, async (conn, transaction, userId) =>
                 {
-                    using (var partitionDb = DatabaseAccess.GetConnection())
-                    using (var transaction = await partitionDb.BeginTransactionAsync(IsolationLevel.ReadUncommitted, cancellationToken))
-                    {
-                        Interlocked.Add(ref totalScores, (ulong)await ScoreProcessor.ProcessUserScoresAsync(userId, RulesetId, db, transaction, cancellationToken));
-                        await transaction.CommitAsync(cancellationToken);
-                    }
+                    Interlocked.Add(ref totalScores, (ulong)await ScoreProcessor.ProcessUserScoresAsync(userId, RulesetId, conn, transaction, cancellationToken));
+                    await transaction.CommitAsync(cancellationToken);
```

Comment:
Is this intentional? It sounds like this would break things because the transaction would still be in use for the next iteration.

Reply:
looks like a merge resolution fail